### PR TITLE
feat: material 0.3

### PR DESCRIPTION
**Pull request by zhuxudong**
material 0.3
- [x] OrbitControl
- [x] FreeControl
- [x] add normalTexture && normalIntensity in `BlinnPhongMaterial`
- [x] add tilingOffset into all materials
- [x] delete `alphaMode`, `doubleSided`
- [x] add `isTransparent`,`renderFace`,`blendMode`
- [x] rename `baseTexture`, `baseColor` 
- [x] 'alphaCutoff' will take effect automatically

### CODE REVIEW

File: zh-cn/resource/custom-material.md [hunk 1]
```diff
@@ -203,11 +203,14 @@ target.destinationColorBlendFactor = target.destinationAlphaBlendFactor = BlendF
 // 操作方式为 src + dst  */
```
Comment (eyworldwide) [inline zh-cn/resource/custom-material.md:203]: Add playground demo link in this doc

File: zh-cn/resource/material.md [hunk 2]
```diff
@@ -23,9 +23,19 @@ const material = new BlinnPhongMaterial(engine);
 material.diffuseColor.r = 0;
 
 // 替换 PBRMaterial，赋予 blinn-phong 材质
-const material = renderer.setSharedMaterial(0, material);
+const material = renderer.setMaterial(material);
 ```
```
Comment (eyworldwide) [inline zh-cn/resource/material.md:27]: Add two playground demo links:
1. How to use PBR Material
2. How to use BlinnPhong Material

File: zh-cn/resource/material.md [hunk 3]
```diff
@@ -8,7 +8,7 @@
 // 获取想要修改的 renderer
 const renderer = entity.getComponent(MeshRenderer);
 // 获取 PBRMaterial
-const material = renderer.getSharedMaterial(0);
+const material = renderer.getMaterial();
```
[the remark this answers is not in the document]
Comment (zhuxudong) [reply zh-cn/resource/material.md:11]: +1

File: zh-cn/resource/custom-material.md (hunk 1 above)
Comment (zhuxudong) [reply zh-cn/resource/custom-material.md:206]: +1

File: zh-cn/resource/material.md (hunk 2 above)
Comment (zhuxudong) [reply zh-cn/resource/material.md:27]: +1

**Review by GuoLei1990 — APPROVED** [review]
+1

**Pull request merged**